### PR TITLE
[FEATURE] Introduce php:staticmethod directive

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -2,7 +2,7 @@ parameters:
 	ignoreErrors:
 		-
 			message: "#^Function phpDocumentor\\\\Guides\\\\DependencyInjection\\\\template not found\\.$#"
-			count: 7
+			count: 8
```
Comment (brotkrueml) [inline phpstan-baseline.neon]: I assume you will get a conflict when the global directive is merged (as it also changes this line).

**Review by brotkrueml — DISMISSED** [review]
Can we add later also an example with php:clasc and staticmethod?

File: phpstan-baseline.neon (hunk 1 above)
Comment (linawolf) [reply phpstan-baseline.neon]: yeah, taking care of it

**Comment by linawolf** [comment]
Yes I will add that test once one of the two directives is merged

**Comment by linawolf** [comment]
waiting for #13

**Review by brotkrueml — DISMISSED** [review]

**Comment by linawolf** [comment]
@brotkrueml added the test with the static method within the class

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**